doc/query.md

```diff
@@ -0,0 +1,545 @@
+# Cloudant Query - Android
+
+Cloudant Query - Android is a Java implementation of [Cloudant Query][1] for Android mobile devices. It is included as a sub-package of [sync-android][2].
```

Comment:
It's in the sync-core jar, so I'd use that here rather than the repo name, which is ambiguous.